pliers/stimuli/vector.py

```diff
@@ -0,0 +1,109 @@
+import numpy as np
+import pandas as pd
+import requests
+import json
+from pliers.utils import attempt_to_import, verify_dependencies
+from pliers.stimuli.base import Stim
+
+
+class VectorStim(Stim):
+    ''' Vector stimulus (1-D numpy array)
+
+    Args:
+         array (np.ndarray, list or pd.Series): Vector of values. Can be list, 
+            array or pandas Series. Gets converted to 1-D numpy array.
+        labels (list): List of labels to which probability values refer, if 
+            that applies.
+        data_dict (dict): If defined, overwrites array and labels. Keys are 
+            passed to labels argument, values are passed to array argument.
+        filename (str): Path to file, if array has to be read from file. 
```

Comment:
I wonder if this stim should have so many options for loading, or if should keep it simple and leave that to the user (they can load, and then create stim)

Reply:
leaving read from and save to tsv option only (reads both from `filename` and `url`).